bsk_rl/envs/general_satellite_tasking/scenario/sat_actions.py

```diff
@@ -173,3 +173,24 @@ def set_action(self, action: Union[int, Target, str]):
             self.prev_action_key = self.image(action, self.prev_action_key)
         else:
             super().set_action(action)
+
+
+class NadirImagingActions(DiscreteSatAction):
+    def __init__(self, *args, **kwargs) -> None:
+        """
+        Discrete action to image nadir.
+        """
+        super().__init__(*args, **kwargs)
+
+        # Sets a static Nadir target
+        self.targets = Target(name="nadir", location=np.array([0, 0, 0.1]), priority=0)
+        self.add_action(self.nadirImage, act_name="nadirImage")
```

Comment:
other actions are named things like `action_desat`. Maybe not the best convention, but let's follow it for consistency